content/articles/spatial_mgrs/index.Rmd

```diff
@@ -0,0 +1,427 @@
+E.g. in `31U ES 1 4` X = 1 refers to the half-open interval [**1**0, 20) km.
+
+## What is the relation to coordinate reference systems (CRSs)?
+
+Projected CRSs combine a map projection (such as 'UTM 31N') with a geodetic datum that relates actual positions on the Earth surface to the (unprojected, ellipsoidal) geodetic coordinate system, which also implies defining the ellipsoid and prime meridian.
+The geodetic datum is a property of the geodetic CRS associated with the projected CRS.
+
+This also means that without a geodetic datum, you can't determine the physical location on the Earth surface that corresponds to a pair of UTM coordinates or to a UTM or MGRS grid reference [^ed50].
+See the [CRS tutorial](../../tutorials/spatial_crs_coding/#coordinate-reference-systems-minimal-background) for more information.
+
+[^ed50]: Older series of Belgian maps distributed by the National Geographical Institute (<https://www.ngi.be>) applied the ED50 geodetic datum (European Datum 1950), which affects the position of coordinates (hence MGRS grid) by about 90 m in E-W direction and about 200 m in N-S direction compared to the ETRS89 and WGS 84 datums. OGC URN identifiers of these datums:\
+[ED50 datum](https://epsg.org/datum_6230/European-Datum-1950.html): `urn:ogc:def:datum:EPSG::6230`\
+[ETRS89 ensemble datum](https://epsg.org/datum_6258/European-Terrestrial-Reference-System-1989-ensemble.html): `urn:ogc:def:ensemble:EPSG::6258`\
+[WGS 84 ensemble datum](https://epsg.org/datum_6326/World-Geodetic-System-1984-ensemble.html): `urn:ogc:def:ensemble:EPSG::6326`.
+
+Although the aim of MGRS is positioning and although it needs a compound CRS for its implementation, MGRS is a geocoding standard, not a map projection or a CRS as it adds complexity on top of a compound CRS.
+
+In Belgium, the most relevant UTM CRSs for UTM zone 31 are:
+
+- `r st_crs("EPSG:25831")$Name` (`EPSG:25831`)
+- `r st_crs("EPSG:32631")$Name` (`EPSG:32631`)
+- `r st_crs("EPSG:23031")$Name` (`EPSG:23031`) which was applied in older topograpic maps
+
+Run `sf::st_crs("EPSG:25831")` in R to inspect the WKT string.
+
+## Distribution of the MGRS grid
+
+The NGA Office of Geomatics [distributes](https://earth-info.nga.mil/index.php?dir=coordsys&action=coordsys) worldwide WGS 84 vector layers of MGRS grid polygons [up to 1 km](https://earth-info.nga.mil/index.php?dir=coordsys&action=mgrs-1km-polyline-dloads) precision.
+
+The UF Geoplan Center provides an [MGRS-data site](http://mgrs-data.org) that serves as a library for MGRS data and information, including [geospatial layers](http://mgrs-data.org/data/).
+
+The match of these layers with any WGS 84 UTM CRS and with the WGS 84 graticule can be easily verified post-hoc.
+
+## Criticisms about the MGRS
+
+Criticisms about the MGRS can be found.
+This mainly has to do with the complexity of the system itself.
```

Comment:
Perhaps in a single sentence: "Criticisms about the MGRS are mainly related to the complexity of the system itself."